src/nxt_array.h

```diff
@@ -9,34 +10,32 @@
 
 
 typedef struct {
-    void      *elts;
-    /* nelts has uint32_t type because it is used most often. */
-    uint32_t  nelts;
-    uint16_t  size;
-    uint16_t  nalloc;
-    nxt_mp_t  *mem_pool;
+    void                *elts;
+    nxt_uint_t          nelts;
+    nxt_uint_t          size;
+    nxt_uint_t          nalloc;
+    nxt_mp_t            *mem_pool;
 } nxt_array_t;
 
 
-nxt_inline void
-nxt_array_init(nxt_array_t *array, nxt_mp_t *mp, size_t size)
-{
-    array->elts = nxt_pointer_to(array, sizeof(nxt_array_t));
-    array->size = size;
-    array->mem_pool = mp;
-}
-
 NXT_EXPORT nxt_array_t *nxt_array_create(nxt_mp_t *mp, nxt_uint_t n,
-    size_t size);
+                                         size_t size);
```

Comment:
Here you add a style issue. The indentation for additional parameter lines in prototype is 4 spaces.

Reply:
I marked it in my notes, I will be careful with the indents, since in my case they are set automatically.

Reply:
I highly recommend you to turn off any automatic indentation that your editor has.  I have yet to see any autoindent feature that I consider actually good.  Maybe the one in `vim(1)`... but not even that.

Check https://nginx.org/en/docs/dev/development_guide.html#code_style for our style guide.

Reply:
> I highly recommend you to turn off any automatic indentation that your editor has. I have yet to see any autoindent feature that I consider actually good. Maybe the one in `vim(1)`... but not even that.
> 
> Check https://nginx.org/en/docs/dev/development_guide.html#code_style for our style guide.

Wow! Really very useful information, thank you for this, since earlier I had to almost intuitively determine what style of code you use.